airbyte-integrations/bases/base-typing-deduping-test/src/main/resources/sqlgenerator/reservedkeywords_inputrecords_raw.jsonl

```diff
@@ -0,0 +1,2 @@
+// A column name that is a reserved keyword should be added for each typing & deduping destination.
+{"_airbyte_raw_id": "b2e0efc4-38a8-47ba-970c-8103f09f08d5", "_airbyte_extracted_at": "2023-01-01T00:00:00Z", "_airbyte_data": {"current_date":  "foo"}}
```

Comment:
nit: add {"JOIN": "foo"} as well.  If this meant to be a tested on all destinations, a reserved word `JOIN` is more common across SQL dialects than `current_date`

Reply:
no harm in having multiple fields, but current_date is actually a pretty common reserved word according to wikipedia https://en.wikipedia.org/wiki/List_of_SQL_reserved_words

<img width="1004" alt="image" src="https://github.com/airbytehq/airbyte/assets/5741425/fa46c670-a32b-42be-9b79-61d0c76a7ffd">

(at the very least, the sql standard defines it as a reserved word)

Reply:
Thanks for the suggestion! I was trying to find a good one to use.